core/src/main/java/org/apache/accumulo/core/client/lexicoder/ListLexicoder.java

```diff
@@ -30,6 +30,8 @@
  * A lexicoder to encode/decode a Java List to/from a byte array where the concatenation of each
  * encoded element sorts lexicographically.
  *
+ * Note: Empty lists are not supported.
+ *
```

Comment:
This could use the `@see` javadoc tag to point to SequenceLexicoder

Also, could make the encode method throw an exception when encoding an empty list.  The exception could have a message saying not support and mention the new encoded.

Reply:
Agreed.